spec/statesman/adapters/active_record_spec.rb

```diff
@@ -355,6 +355,18 @@
     end
   end
 
+  it "resets last with #reload" do
+    model.save!
+    ActiveRecord::Base.transaction do
+      model.state_machine.transition_to!(:succeeded)
+      model.state_machine.current_state
```

Comment:
Do we need this line?

Reply:
Yes, it caches the value to the `@last_transition` instance variable. I can add a comment and expectation if you think it's better to see it there.

Reply:
Nono that's fine, I was being silly haha. I assumed it would be that but didn't see anything with a very quick cursory glance.